opentelemetry-sdk/src/opentelemetry/sdk/util/__init__.pyi

```diff
@@ -0,0 +1,71 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
Do import for `__init__.pyi` files work?

Reply:
What do you mean? The pyi don't effect anything at runtime

Reply:
I meant the `from typing import` statements below, I was just curious what an import does in a `.pyi` file